nix/sources.json

```diff
@@ -34,16 +34,16 @@
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "ic-hs": {
-        "branch": "master",
+        "branch": "mraszyk/periodic-system-tasks",
```

Comment:
This needs to be merged first!

Reply:
I've merged the MR.

Reply:
thanks! 80a8b03665e405a2bd653c988970ba7e47ed12ab